src/Validation/NgenServicingAttributes.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) Andrew Arnott. All rights reserved.
+// Licensed under the Ms-PL license. See LICENSE.txt file in the project root for full license information.
+
+#if NETSTANDARD1_3
+namespace System.Runtime
+{
+    [AttributeUsage(AttributeTargets.Assembly, Inherited = false)]
+    public sealed class AssemblyTargetedPatchBandAttribute : Attribute
```

Comment:
If we keep this (which we needn't if we drop netstandard1.3 support), let's make the attributes internal. They should still "work", but the Validation assembly has no business making these types public for others' use.